### PR TITLE
Add G12 FAQ

### PR DESCRIPTION
**Pull request by suehighmoor**

### CODE REVIEW

File: articles/other/other-faq-g12.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+toc_title: G-Cloud 12 enhancements and additions
+toc_fullpath: FAQs/other-faq-g12.md
+toc_mdlink: other-faq-g12.md
+---
+
+# G-Cloud 12 - Our enhancements and additions
+
+## Summary
+
+### What is the UKCloud theme for G-Cloud 12?
```
Comment (vrush20) [inline articles/other/other-faq-g12.md:22]: Should be consistent G12 or G-Cloud 12

File: articles/other/other-faq-g12.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+### What is Neustar DDoS Protection from UKCloud?
+
+UKCloud's internet-connected Assured OFFICIAL platform already has Neustar DDoS mitigation baked in, however this mitigation is aimed at protecting the entire platform and is based on UKCloud's aggregate traffic profile. Neustar's UltraDDos Protect and UltraWAF are tailored to your own organisation's traffic patterns, providing you with powerful analytics, top-tier personalised DDoS mitigation and layer 7 protection that enables you to suppress threats before they become attacks. Although previously offered as a service option, this is now a separate service with its own Service Definition.
+
+## Changes to existing services
+
+For information about any of the updated services listed below, see [*UKCloud Service Definitions and Pricing Guide*](https://docs.ukcloud.com/articles/other/other-ref-service-definitions.html).
+
+### What's new in the UKCloud for OpenShift service?
```
Comment (vrush20) [inline articles/other/other-faq-g12.md:92]: this service now offers customers on OCPv4 or higher, the option for...

**Review by vrush20 — APPROVED** [review]
Couple of minor comments

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

File: articles/other/other-faq-g12.md (hunk 1 above)
Comment (suehighmoor) [reply articles/other/other-faq-g12.md:22]: Changed to G-Cloud 12 throughout

File: articles/other/other-faq-g12.md (hunk 2 above)
Comment (suehighmoor) [reply articles/other/other-faq-g12.md:92]: Done

**Pull request merged**